templates/js/link-unfurling/.webappignore

```diff
@@ -0,0 +1,26 @@
+.webappignore
+.fx
+.deployment
+.localConfigs
+.vscode
+*.js.map
+*.ts.map
+*.ts
+.git*
+.tsbuildinfo
+CHANGELOG.md
+readme.md
+local.settings.json
+test
+tsconfig.json
+.DS_Store
+teamsapp.yml
+teamsapp.*.yml
+/env/
+/script/
+/node_modules/.bin
+/node_modules/ts-node
+/node_modules/typescript
+/appPackage/
+/infra/
+/teamsfx/
```

Comment:
Your project doesn't contain the teamsfx and the script folder. it can be removed.